code/game/gamemodes/cult/runes.dm

```diff
@@ -490,6 +490,23 @@ var/list/teleport_runes = list()
 			M << "<span class='warning'>Nar-Sie is already on this plane!</span>"
 		log_game("Summon Nar-Sie rune failed - already summoned")
 		return
+		
+	for(var/M in invokers)
+		if(istype(M, /mob/living/carbon/human))
+			if(/datum/antagonist/ghostcultist in M.antag_datums)
```

Comment:
You will need to use M.has_antag_datum(path), since antag_datums is a list of already initialized datums.